src/dex-ui-components/base/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,108 @@
+import { Stepper, Step, StepLabel } from "@material-ui/core";
+import { CheckCircleIcon, createIcon } from "@chakra-ui/icons";
+import { Color } from "../../themes";
+import { Flex } from "@chakra-ui/layout";
+import { Box, Text } from "@chakra-ui/react";
+import { StepperStyles } from "./StepperStyles";
+import { StepperState, ProposalStateIcon } from "./types";
+
+const DisabledIcon = createIcon({
```

Comment:
We can leave these Icons here for now but we may want to move them into the dex-ui Icons folder once we create the official DEX Component Library.

Reply:
Done, I have moved the icons under dex-ui-components/base/Icons folder.

Reply:
Also the reduced the size of `Icons` to match the Wireframe